tests/raop/test_raop_functional.py

```diff
@@ -135,6 +136,20 @@ async def test_stream_complete_legacy_auth(
     assert raop_state.auth_setup_performed == require_auth
     assert audio_matches(raop_state.raw_audio, frames=10)
 
+@pytest.mark.parametrize(
+    "raop_properties,raop_server_password,raop_client_password",
+    [({"et": "0"}, "test", "test"), ({"et": "0"}, None, None)],
+)
+async def test_stream_with_password(
+    raop_client, raop_state, raop_usecase, raop_server_password, raop_client_password
+):
+    raop_usecase.password(raop_server_password)
+
+    raop_service = raop_client.get_service(Protocol.RAOP)
```

Comment:
The `raop_conf` fixture holds the config, add that and get the service from there instead.

https://github.com/postlund/pyatv/blob/809c5a384cd7762410919d1e6cdc53f3e973f93f/tests/raop/conftest.py#L33